lab_1_keywords_tfidf/start.py

```diff
@@ -1,14 +1,15 @@
 """
 Frequency-driven keyword extraction starter
 """
-import json
 from pathlib import Path
+import json
+import main
 
 
 if __name__ == "__main__":
 
     # finding paths to the necessary utils
-    PROJECT_ROOT = Path(__file__).parent
+    PROJECT_ROOT = Path('new').parent
```

Comment:
Could you explain this change?

Reply:
All files for this work are stored in the folder 'new'

Reply:
does it fail when you leave this pre-written part of the code as is? 
if not, I suggest you do not change it